src/services/LogFileManager/decodeUtils.ts

```
@@ -0,0 +1,156 @@
+    for (const entry of FILE_TYPE_DEFINITIONS) {
+        const matchingExtension = entry.extensions.find((ext) => fileName.endsWith(ext));
+        if (matchingExtension) {
+            try {
+                return {
+                    decoder: await entry.DecoderFactory.create(fileData, decoderOptions),
+                    matchingExtension: matchingExtension,
+                    fileTypeDef: entry,
+                };
+            } catch (e) {
+                console.warn(`File extension matches ${entry.name},` +
+                        "but decoder creation failed.", e);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)